bitnami/metrics-server/Chart.yaml

```
@@ -23,4 +23,4 @@ name: metrics-server
 sources:
   - https://github.com/bitnami/bitnami-docker-metrics-server
   - https://github.com/kubernetes-incubator/metrics-server
-version: 5.6.0
+version: 5.6.1
```

Please replace this with a minor version bump
```suggestion
version: 5.7.0
```

---

👍